templates/partial/_featured-charms.html

```diff
@@ -14,7 +14,7 @@ <h3 class="p-card__title p-heading--4 u-no-margin--bottom package-card-title">{{
     </div>
 
     <div class="p-card__content">
-      <p><small class="package-card-summary">{{ charm.result.summary }}</small></p>
+      <p><small class="package-card-summary">{{ charm.result.summary[:90].strip() }}{% if charm.result.summary[:90]|length == 90 %}&hellip;{% endif %}</small></p>
```

Comment:
In the template you could just do:
```suggestion
      <p><small class="package-card-summary">{{ charm.result.summary|truncate(90) }}</small></p>
```